community_supplied/Interface/interface-kpis-additional.playbook

```diff
@@ -0,0 +1,16 @@
+/*
+ * Playbook contains multiple rules which monitor interfaces and notifies when
+ * anomalies are found.
+ * 
+ * 1) Rule check-ethernet-pause-frame-counters-netconf, detects interface ethernet 
+ *    pause frame counters and notifies when anomalies are found.
+ * 2) Rule check-if-mtu-netconf, Detects interface MTU values and notifies when 
+ *    anomalies are found.
+ */
+ healthbot {
+    playbook interface-kpis-additional-playbook {
```

Comment:
change playbook name from "interface-kpis-additional-playbook" to "interface-additional-kpis-playbook"

Reply:
changed the playbook name to "interface-additional-kpis-playbook"